rpc/rpc-interfaces.go

```diff
@@ -12,5 +12,5 @@ type Client interface {
 
 // Server describes the functions an RPC Server performs
 type Server interface {
-	Handle(string, func([]byte) ([]byte, error))
```

Comment:
this means that nothing outside of the rpc package can implement this, no? because there'll be a compile error when another package has `..., handler rpc.messageHandler)` in it.